controls/RHEL-07-021010.rb

```diff
@@ -35,8 +35,13 @@
 If a file system found in “/etc/fstab” refers to the user home directory file system and it does not have the “nosetuid” option set, this is a finding.'
 
 # START_DESCRIBE RHEL-07-021010
-  describe file('') do
-    it { should match // }
+  fstab_lines = file('/etc/fstab').content.split("\n")
```

Comment:
I am getting an error on this line when I run inspec check on this control. Not sure what the issue is.

```
inspec check inspec-stig-rhel7/controls//RHEL-07-021010.rb
inspec-stig-rhel7/controls//RHEL-07-021010.rb:38:in `block in load_with_context': undefined method `split' for nil:NilClass (NoMethodError)
	from /opt/inspec/embedded/lib/ruby/gems/2.3.0/gems/inspec-1.7.1/lib/inspec/rule.rb:51:in `instance_eval'
	from /opt/inspec/embedded/lib/ruby/gems/2.3.0/gems/inspec-1.7.1/lib/inspec/rule.rb:51:in `initialize'
	from /opt/inspec/embedded/lib/ruby/gems/2.3.0/gems/inspec-1.7.1/lib/inspec/control_eval_context.rb:73:in `new'
	from /opt/inspec/embedded/lib/ruby/gems/2.3.0/gems/inspec-1.7.1/lib/inspec/control_eval_context.rb:73:in `block (2 levels) in create'
	from inspec-stig-rhel7/controls//RHEL-07-021010.rb:8:in `load_with_context'
	from /opt/inspec/embedded/lib/ruby/gems/2.3.0/gems/inspec-1.7.1/lib/inspec/profile_context.rb:146:in `instance_eval'
	from /opt/inspec/embedded/lib/ruby/gems/2.3.0/gems/inspec-1.7.1/lib/inspec/profile_context.rb:146:in `load_with_context'
	from /opt/inspec/embedded/lib/ruby/gems/2.3.0/gems/inspec-1.7.1/lib/inspec/profile_context.rb:130:in `load_control_file'
	from /opt/inspec/embedded/lib/ruby/gems/2.3.0/gems/inspec-1.7.1/lib/inspec/profile.rb:144:in `block in collect_tests'
	from /opt/inspec/embedded/lib/ruby/gems/2.3.0/gems/inspec-1.7.1/lib/inspec/profile.rb:141:in `each'
	from /opt/inspec/embedde....
```